test/Wallet.test.ts

```diff
@@ -97,6 +98,24 @@ describe("Wallet", () => {
   // broadcastRawTx
   //
 
+  describe("broadcastRawTx()", () => {
```

Comment:
I think `should be successfull if the network provider returns the valid hex.` is needed.